google/cloud/forseti/notifier/notifiers/base_notification.py

```diff
@@ -89,3 +89,16 @@ def _get_output_filename(self, filename_template):
         output_filename = filename_template.format(
             self.resource, self.cycle_timestamp, output_timestamp)
         return output_filename
+
+    @classmethod
+    def check_data_format(cls, data_format):
+        """Raise `InvalidDataFormatError` unless `data_format` is supported.
+
+        Args:
+            data_format (string): should be either 'csv' or 'json'
+
+        Raises:
+            InvalidDataFormatError: if not valid
+        """
+        if data_format not in cls.supported_data_formats:
+            raise InvalidDataFormatError('GCS uploader', data_format)
```

Comment:
If this is a base method that could be used by others, should you hardcode 'GCS uploader' here?

Reply:
Fixed.